openapi.yaml

```diff
@@ -46,15 +46,19 @@ paths:
           type: string
           required: false
           description: Query string to be used when searching through the contents of each note
```

Comment:
Can you also mention what does `-` stand for? https://jsonapi.org/format/#fetching-sorting